flux-refineck/src/fold_unfold.rs

```diff
@@ -315,6 +316,9 @@ impl<'a, 'tcx, M: Mode> FoldUnfoldAnalysis<'a, 'tcx, M> {
             TerminatorKind::Goto { target } => {
                 self.goto(bb, *target, env)?;
             }
+            TerminatorKind::Yield { resume, .. } => {
+                self.goto(bb, *resume, env)?;
```

Comment:
You should do a `M::projection` of the corresponding place here

Reply:
@nilehmann I added this, but please check if its the right thing, as I don't know what `M::projection` is supposed to do!

Reply:
Yes, this looks good